.github/workflows/call.check-query-schema-against-subgraph.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         run: |
           yarn build:contracts-truffle
           yarn build:contracts-hardhat
+          yarn build:typechain
```

Comment:
not a blocking comment, but having the same build:* sequence across several places tickles DRY principle. something to tackle later.

Reply:
will resolve this in the same PR along w/ squashed commit - just does yarn build instead